pybloqs/server/block/life_loading.py

```diff
@@ -0,0 +1,137 @@
+import base64
+from typing import Iterator, Optional
+
+import pybloqs
+from pybloqs.html import append_to
+from pybloqs.static import Css, JScript
+
+BOX_SVG = """
+<svg width="8mm" height="8mm" viewBox="0 0 7 7" version="1.1" id="svg5" xmlns="http://www.w3.org/2000/svg" xmlns:svg="http://www.w3.org/2000/svg">
+  <rect
+    style="fill:#000000;stroke-width:1.0;fill-opacity:0;stroke-miterlimit:4;stroke-dasharray:none;stroke:#C6D9F1;stroke-opacity:1"
+    id="rec t84"
+    width="7"
+    height="7"
+    x="0"
+    y="0" />
+</svg>
+"""
+
+CSS_STRING = f"""
+.pybloqs_life {{
+  position: relative;
+  height: 200px;
+  overflow: hidden;
+}}
+.pybloqs_life_background {{
+  position: absolute;
+  image-rendering: pixelated;
+  top: 50%;
+  left: 50%;
+  width: 100%;
+  height: 100%;
+  transform: translate(-50%, -50%);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}}
+.pybloqs_life_background > * {{
+  position: absolute;
+  flex-shrink: 0;
+  object-fit: cover;
+  height: 100%;
+  width: 100%;
+  aspect-ratio: 1;
+}}
+.pybloqs_life_background>.grid {{
+  background-image: url('data:image/svg+xml;base64,{base64.b64encode(BOX_SVG.encode()).decode()}');
+  background-size: 2%;
+  height: unset;
+}}
+.pybloqs_life_foreground {{
+  position: relative;
+  z-index: 1;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+  background: #ffffffa0;
+  font-weight: bold;
+  font-size: 2rem;
+}}
+"""
+
+# This magic string is a compiled WASM program that exports a buffer called
+# `BUFFER` of 50 by 50 pixels and a function called `go` which iterates
+# Conway's game of life on the buffer.
```

Comment:
and what is that for? is it a loading screen?

Reply:
Yes its a loading screen. See the section "loading screen" in the PR description